backend/posts/views.py

```diff
@@ -40,9 +40,3 @@ def create(self, request, *args, **kwargs):
         serialized_data = serializer.data
 
         return Response(serialized_data, status=status.HTTP_201_CREATED)
-
-    def retrieve(self, request, *args, **kwargs):
-        instance = self.get_object()
-        serializer = PostRetrieveSerializer(instance)
-
-        return Response(serializer.data)
```

Comment:
Waarom moet deze weg? Dit heb ik zelf toegevoegd voor het ophalen van de post info voor de frontend namelijk.

Reply:
Omdat we nog maar 1 serializer hebben. Het enigste wat je hier doet is die aanroepen, maar omdat we er nu 1 hebben is dat de serializer_class.